addons/point_of_sale/tests/test_pos_products_with_tax.py

```diff
@@ -300,3 +300,183 @@ def _after_closing_cb():
                 'bank_payments': [],
             },
         })
+
+    def test_pos_create_correct_account_move(self):
+        """ Test for orders with global rounding disabled
+
+        Orders
+        ======
+        +---------+----------+-----------+----------+------+----------+------------------+--------+
+        | order   | payments | invoiced? | product  | qty  |  untaxed | tax              |  total |
+        +---------+----------+-----------+----------+------+----------+------------------+--------+
+        | order 1 | cash     | no        | product1 |    1 |     10.0 |  2.10            |  12.10 |
+        |         |          |           | product2 |   -1 |     -5.0 | -1.05            |  -6.05 |
+        +---------+----------+-----------+----------+------+----------+------------------+--------+
+        """
+        tax_21_incl = self.taxes['tax21']
+        product1 = self.create_product(
+            name='Product 1',
+            category=self.categ_basic,
+            lst_price=12.10,
+            tax_ids=tax_21_incl.ids,
+        )
+        product2 = self.create_product(
+            name='Product 2',
+            category=self.categ_basic,
+            lst_price=6.05,
+            tax_ids=tax_21_incl.ids,
+        )
+        self.open_new_session()
+        self.env['pos.order'].create_from_ui([self.create_ui_order_data([
+            (product1, 1),
+            (product2, -1),
+        ])])
+        self.pos_session.action_pos_session_validate()
+
+        lines = self.pos_session.move_id.line_ids.sorted('balance')
+        self.assertEqual(2, len(lines.filtered(lambda l: l.tax_ids)), "Taxes should have been set on 2 lines")
+        self.assertEqual(4, len(lines.filtered(lambda l: l.tax_tag_ids)), "Tags should have been set on 4 lines")
+        self.assertRecordValues(lines, [
+            {'account_id': self.sale_account.id,           'balance': -10.0, 'tax_ids': tax_21_incl.ids, 'tax_tag_ids': self.tax_tag_invoice_base.ids},
+            {'account_id': self.tax_received_account.id,   'balance': -2.10, 'tax_ids': [],              'tax_tag_ids': self.tax_tag_invoice_tax.ids},
+            {'account_id': self.tax_received_account.id,   'balance':  1.05, 'tax_ids': [],              'tax_tag_ids': self.tax_tag_refund_tax.ids},
+            {'account_id': self.sale_account.id,           'balance':  5.00, 'tax_ids': tax_21_incl.ids, 'tax_tag_ids': self.tax_tag_refund_base.ids},
+            {'account_id': self.cash_pm1.receivable_account_id.id, 'balance':  6.05, 'tax_ids': [],              'tax_tag_ids': []},
```

Comment:
Please use `# pylint: disable=bad-whitespace` to ignore the linting error on CI/style

Also check the indentation on this line

Reply:
Thanks! :)